src/api/db/migrate/20160824132643_fix_bs_request_counter.rb

```diff
@@ -0,0 +1,10 @@
+class FixBsRequestCounter < ActiveRecord::Migration
+  def self.change
+    # BsRequestCounter is not set correctly
+    # Introduced with 20160321105300_request_counter.rb
+    # See https://github.com/openSUSE/open-build-service/issues/2068
+    last_request = BsRequest.all.order(:number).last
```

Comment:
[.reorder](http://apidock.com/rails/ActiveRecord/QueryMethods/reorder) sounds more appropriate and please pluck the the attribute...